includes/Story_Post_Type.php

```diff
@@ -559,6 +559,7 @@ public function get_editor_settings() {
 				'version'          => WEBSTORIES_VERSION,
 				'encodeMarkup'     => $this->decoder->supports_decoding(),
 				'metaBoxes'        => $this->meta_boxes->get_meta_boxes_per_location(),
+				'ffmpegCoreUrl'    => trailingslashit( WEBSTORIES_CDN_URL ) . 'js/@ffmpeg/core@0.8.5/dist/ffmpeg-core.js',
```

Comment:
Why is this loaded via the web stories cdn url?

Reply:
Bundling it with the plugin is not an option because the files are quite large.

By using our CDN instead we'd have more control over this vs. using unpkg.com (the default).

We can see how well that works, perhaps it's easier to just stick with unpkg in the end.